packages/mco/components/empty-state-page/empty-page.tsx

```diff
@@ -34,7 +36,7 @@ const EmptyPage: React.FC<EmptyPageProps> = (props) => {
     <EmptyState variant={EmptyStateVariant.lg}>
       <EmptyStateHeader
         titleText={<>{title}</>}
-        icon={<EmptyStateIcon icon={CubesIcon} />}
+        icon={<EmptyStateIcon icon={EmptyPageIcon || CubesIcon} />}
```

Comment:
nits:
where `EmptyIcon = CubesIcon` (by default) and we can pass any custom icon value, here that custom value will be `EmptyIcon = EmptyPageIcon`...
```suggestion
        icon={<EmptyStateIcon icon={EmptyIcon || CubesIcon} />}
```

Reply:
ack